deps/build.jl

```diff
@@ -66,7 +64,8 @@ provides(BuildProcess, (@build_steps begin
         FileRule(joinpath(libdir(libfftw), "libfftw." * Libdl.dlext), @build_steps begin
             CreateDirectory(libdir(libfftw))
             `$(joinpath(".", "configure")) $general_config $fftw_config $fftw_enable_single`
-            `$MAKE`
+            `$MAKE_CMD`
+            `$MAKE_CMD install`
```

Comment:
I... don't know. I was just trying to match the logic as I understood it in the makefile in base.